start-demo script fails on new Ubuntu 16.04

I've set up a new machine to test out Kapua.  It has a completely new installation of Ubuntu 16.04 LTS with vagrant and virtualbox installed out of the repositories:

```
joe@jjm-w110er:~$ apt search vagrant | grep installed

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

vagrant/xenial,xenial,now 1.8.1+dfsg-1 all [installed]
joe@jjm-w110er:~$ apt search virtualbox | grep installed

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

unity-scope-virtualbox/xenial,xenial,now 0.1+13.10.20130723-0ubuntu1 all [installed]
vagrant/xenial,xenial,now 1.8.1+dfsg-1 all [installed]
virtualbox/xenial-updates,now 5.0.32-dfsg-0ubuntu1.16.04.2 amd64 [installed]
virtualbox-dkms/xenial-updates,xenial-updates,now 5.0.32-dfsg-0ubuntu1.16.04.2 all [installed,automatic]
virtualbox-qt/xenial-updates,now 5.0.32-dfsg-0ubuntu1.16.04.2 amd64 [installed,automatic]
joe@jjm-w110er:~$ apt search eclipse | grep installed

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

eclipse/xenial,xenial,now 3.8.1-8 all [installed]
eclipse-jdt/xenial,xenial,now 3.8.1-8 all [installed,automatic]
eclipse-pde/xenial,now 3.8.1-8 amd64 [installed,automatic]
eclipse-platform/xenial,now 3.8.1-8 amd64 [installed,automatic]
eclipse-platform-data/xenial,xenial,now 3.8.1-8 all [installed,automatic]
eclipse-rcp/xenial,now 3.8.1-8 amd64 [installed,automatic]
libecj-java/xenial-updates,xenial-updates,now 3.10.1-2ubuntu1 all [installed,automatic]
libeclipse-aether-java/xenial,xenial,now 1.0.2-1 all [installed,automatic]
libequinox-osgi-java/xenial,xenial,now 3.8.1-8 all [installed,automatic]
libgeronimo-jpa-2.0-spec-java/xenial,xenial,now 1.1-2 all [installed,automatic]
libxbean-java/xenial,xenial,now 4.5-1 all [installed,automatic]
joe@jjm-w110er:~$ apt search maven | grep installed

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

libapache-pom-java/xenial,xenial,now 10-2build1 all [installed,automatic]
libcommons-parent-java/xenial,xenial,now 39-3 all [installed,automatic]
libdoxia-core-java/xenial,xenial,now 1.1.4-3ubuntu1 all [installed,automatic]
libeclipse-aether-java/xenial,xenial,now 1.0.2-1 all [installed,automatic]
libmaven-parent-java/xenial,xenial,now 21-2 all [installed,automatic]
libmaven2-core-java/xenial,xenial,now 2.2.1-24 all [installed,automatic]
libmaven3-core-java/xenial,xenial,now 3.3.9-3 all [installed,automatic]
libplexus-cipher-java/xenial,xenial,now 1.7-1 all [installed,automatic]
libplexus-component-metadata-java/xenial,xenial,now 1.6-2 all [installed,automatic]
libplexus-interpolation-java/xenial,xenial,now 1.21-1 all [installed,automatic]
libplexus-io-java/xenial,xenial,now 1.0~alpha5-2 all [installed,automatic]
libplexus-sec-dispatcher-java/xenial,xenial,now 1.3.1-6 all [installed,automatic]
libsisu-plexus-java/xenial,xenial,now 0.3.2-1 all [installed,automatic]
libwagon-java/xenial,xenial,now 1.0.0-7 all [installed,automatic]
libwagon2-java/xenial,xenial,now 2.10-3 all [installed,automatic]
maven/xenial,xenial,now 3.3.9-3 all [installed]
```
Following the instructions at http://www.eclipse.org/kapua/getting-started.php I cloned the Kapua git repository (sitting on the default `develop` branch) and ran the `start-demo.sh` script.  It fails in a few places, but key one appears to be here:

```
[ERROR] Failed to execute goal org.codehaus.mojo:sql-maven-plugin:1.5:execute (create-schema) on project kapua-commons: Connection is broken: "java.net.SocketTimeoutException: connect timed out: 192.168.33.10:3306" [90067-192] -> [Help 1]
```
and appears to be due to there not being a route for 192.168.33.10 inside the vagrant VM

I captured the full log here: https://gist.github.com/joeythesaint/c33a33a9236388583d16bfd257b8a48b

Can anyone provide any pointers to what's going wrong here?

--- a/commons/pom.xml
+++ b/commons/pom.xml
@@ -72,35 +72,5 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
- 
-	<profiles>
-		<profile>
-			<id>sql</id>
-			<build>
-				<plugins>
-					<plugin>
-						<groupId>org.codehaus.mojo</groupId>
-						<artifactId>sql-maven-plugin</artifactId>
-						<executions>
-							<execution>
-								<id>create-schema</id>
-								<phase>process-test-resources</phase>
-								<goals>
-									<goal>execute</goal>
-								</goals>
-								<configuration>
-									<autocommit>true</autocommit>
-									<srcFiles>
-										<srcFile>src/main/sql/H2/sys_configuration_drop.sql</srcFile>
-                                        <srcFile>src/main/resources/liquibase/configuration.sql</srcFile>
-                                    </srcFiles>
-								</configuration>
-							</execution>
-						</executions>
-					</plugin>
-				</plugins>
-			</build>
-		</profile>
-	</profiles>
         
 </project>

--- a/dev-tools/src/main/bin/start-demo.sh
+++ b/dev-tools/src/main/bin/start-demo.sh
@@ -1,6 +1,6 @@
 export KAPUA_GIT_ROOT="$(pwd)"
 export ACTIVEMQ_VERSION=5.14.0
-export TOMCAT_VERSION=8.0.38
+export TOMCAT_VERSION=8.0.41
 
 cd ../vagrant
 
@@ -22,7 +22,7 @@ echo 'creating demo vagrant machine'
 
 cd $KAPUA_GIT_ROOT/../../../../
 
-mvn clean install -Psql -Pdeploy
+mvn clean install -P deploy -D skipTests
 
 cd dev-tools/src/main/vagrant
 

--- a/dev-tools/src/main/vagrant/baseBox/baseBox-Vagrantfile
+++ b/dev-tools/src/main/vagrant/baseBox/baseBox-Vagrantfile
@@ -22,7 +22,7 @@ Vagrant.configure(2) do |config|
 	 export ELASTICSEARCH_VERSION="2.3.4"
 	 export H2DB_VERSION="1.4.192"
 	 export ACTIVE_MQ_VERSION="5.14.0"
-	 export TOMCAT_VERSION="8.0.38"
+	 export TOMCAT_VERSION="8.0.41"
 
      # update
      sudo apt-get update -y

--- a/dev-tools/src/main/vagrant/demo/demo-Vagrantfile
+++ b/dev-tools/src/main/vagrant/demo/demo-Vagrantfile
@@ -54,7 +54,7 @@ Vagrant.configure("2") do |config|
 	 export ELASTICSEARCH_VERSION="2.3.4"
 	 export H2DB_VERSION="1.4.192"
 	 export ACTIVE_MQ_VERSION="5.14.0"
-	 export TOMCAT_VERSION="8.0.38"
+	 export TOMCAT_VERSION="8.0.41"
 
 	 ###########################
      ### H2 database startup ###

--- a/dev-tools/src/main/vagrant/demo/deployScript/deploy-broker.sh
+++ b/dev-tools/src/main/vagrant/demo/deployScript/deploy-broker.sh
@@ -18,8 +18,9 @@ vagrant ssh -c "echo 'deploying the Kapua broker'
 	echo 'deleting old Kapua runtime dependency'
 	find lib/extra ! -name 'mqtt-client*.jar' -type f -exec rm -f {} +
 	echo 'copying Kapua runtime dependency'
-	sudo cp /kapua/assembly/target/broker_dependency/* lib/extra
-	echo 'copying Kapua arctifact'
+    sudo cp /kapua/assembly/target/broker_dependency/* lib/extra
+	for name in \$(ls /kapua/broker-core/target/dependency/ | grep -Ev 'jaxb-|activemq-|kapua-'); do echo copying from /kapua/broker-core/target/dependency/\$name lib/extra/\$name; sudo cp /kapua/broker-core/target/dependency/\$name lib/extra/\$name; done;
+	echo 'copying Kapua artifact'
 	sudo cp /kapua/broker-core/target/kapua-*.jar lib/extra
 	sudo cp /kapua/commons/target/kapua-*.jar lib/extra
 	sudo cp /kapua/service/device/api/target/kapua-*.jar lib/extra
@@ -46,6 +47,7 @@ vagrant ssh -c "echo 'deploying the Kapua broker'
 	sudo cp /kapua/service/device/registry/internal/target/kapua-*.jar lib/extra
 	sudo cp /kapua/service/idgenerator/api/target/kapua-*.jar lib/extra
 	sudo cp /kapua/service/idgenerator/sequence/target/kapua-*.jar lib/extra
+	sudo cp /kapua/service/liquibase/target/kapua-*.jar lib/extra
 	sudo cp /kapua/service/security/authentication/api/target/kapua-*.jar lib/extra
 	sudo cp /kapua/service/security/authorization/api/target/kapua-*.jar lib/extra
 	sudo cp /kapua/service/security/shiro/target/kapua-*.jar lib/extra
@@ -59,6 +61,7 @@ vagrant ssh -c "echo 'deploying the Kapua broker'
 	sudo cp /kapua/transport/jms/target/kapua-*.jar lib/extra
 	sudo cp /kapua/transport/mqtt/target/kapua-*.jar lib/extra
 	echo 'copying Kapua configuration'
-	sudo cp /kapua/assembly/src/main/resources/conf/broker/activemq.xml conf/
+	sudo cp /kapua/assembly/src/main/resources/conf/broker/activemq.xml conf/ 
+	sudo cp /kapua/assembly/src/main/resources/conf/broker/camel.xml conf/
 	cd ..
 	sudo chown -R vagrant:vagrant apache-activemq-${ACTIVEMQ_VERSION}"

--- a/dev-tools/src/main/vagrant/develop/develop-Vagrantfile
+++ b/dev-tools/src/main/vagrant/develop/develop-Vagrantfile
@@ -53,7 +53,7 @@ Vagrant.configure("2") do |config|
 	 export ELASTICSEARCH_VERSION="2.3.4"
 	 export H2DB_VERSION="1.4.192"
 	 export ACTIVEMQ_VERSION="5.14.0"
-	 export TOMCAT_VERSION="8.0.38"
+	 export TOMCAT_VERSION="8.0.41"
 	 export KAPUA_VERSION="0.2.0-SNAPSHOT"
 
 	 #########################

--- a/dev-tools/src/main/vagrant/develop/script/broker/update-kapua-jars-cfg.sh
+++ b/dev-tools/src/main/vagrant/develop/script/broker/update-kapua-jars-cfg.sh
@@ -14,7 +14,7 @@ echo 'cleanup the symbolic links to Kapua jars'
 for name in $(find lib/extra -type l); do echo remove symbolic link from ./$name; rm ./$name; done;
 echo 'create the symbolic links to Kapua jars'
 echo '===> copy dependencies for broker core'
-for name in $(ls /kapua/broker-core/target/dependency/ | grep -Ev 'jaxb-|activemq-|kapua-'); do echo create symbolic link from ./lib/extra$name /kapua/broker-core/target/dependency/$name; ln -s /kapua/broker-core/target/dependency/$name ./lib/extra/$name; done;
+for name in $(ls /kapua/broker-core/target/dependency/ | grep -Ev 'jaxb-|activemq-|kapua-'); do echo create symbolic link from ./lib/extra/$name /kapua/broker-core/target/dependency/$name; ln -s /kapua/broker-core/target/dependency/$name ./lib/extra/$name; done;
 echo '===> copy dependencies for locator/guice'
 for name in $(ls /kapua/locator/guice/target/dependency/ | grep -Ev 'jaxb-|activemq-|kapua-'); do echo create symbolic link from ./lib/extra/$name /kapua/locator/guice/target/dependency/$name; ln -s /kapua/locator/guice/target/dependency/$name ./lib/extra/$name; done;
 echo '===> copy kapua modules'

